msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CodesignTaskBase.cs

```diff
@@ -98,6 +100,9 @@ string GenerateCommandLineArguments (ITaskItem item)
 			if (UseHardenedRuntime)
 				args.Add ("-o runtime");
 
+			if (UseSecureTimestamp)
+				args.Add ("--timestamp");
```

Comment:
how old is that (and `-o runtime`) option(s) ?

what happen (any cost at build/run time) if we always enabled them ?

Reply:
We're using `--timestamp=none` for XI because it's faster: https://bugzilla.xamarin.com/show_bug.cgi?id=51298#c3, which presumably means that just `--timestamp` may slow the build down somewhat.

Reply:
@spouliot `-o runtime` means "enable hardened runtime", which will crash your app hard unless you have the correct entitlements. It is unsafe to add it unconditionally.

Let me look into if you can notary with --timestamp=none, the docs suggest you need --timestamp. Maybe we should do none when not hardened?

Reply:
@rolfbjarne make sense as this is usually a 3rd party timestamp service. It's likely a problem too if you're not connected to the internet.

@chamons ok, if it crash then it can't be default (I thought it was more an advertisement that hardened was supported, irrespective of the entitlement).

Reply:
If you --timestamp=none you get `"The signature does not include a secure timestamp."` when you notarize.